bridges/modules/xcm-bridge-hub/src/lib.rs

```diff
@@ -213,9 +213,8 @@ pub mod pallet {
 		type DestinationVersion: GetVersion;
 
 		/// The origin that is allowed to call privileged operations on the pallet, e.g. open/close
-		/// bridge for location that coresponds to `Self::BridgeOriginAccountIdConverter` and
-		/// `Self::BridgedNetwork`.
-		type AdminOrigin: EnsureOrigin<<Self as SystemConfig>::RuntimeOrigin>;
+		/// bridge for locations.
+		type ForceOrigin: EnsureOrigin<<Self as SystemConfig>::RuntimeOrigin>;
```

Comment:
well, I just see `PriviledgedOrigin` only on one place:
```
impl validator_manager::Config for Runtime {
	type RuntimeEvent = RuntimeEvent;
	type PrivilegedOrigin = EnsureRoot<AccountId>;
}
```

usually we have `AdminOrigin` or `ForceOrigin` and actually, this is not used here, but I am planning to add `force_open_bridge` and `force_close_bridge` for root or governance scenarios, so that's why I tried to align it with other pallets.

Reply:
Oh, ok, makes sense then to name it `ForceOrigin`